docs/core/entity/lawn-mower.md

```
@@ -36,6 +37,7 @@ and are combined using the bitwise or (`|`) operator.
 | `START_MOWING` | The lawn mower supports the start mowing command.    |
 | `PAUSE`        | The lawn mower supports pausing the current task.    |
 | `DOCK`         | The lawn mower supports the return to dock command.  |
+| `STOP`         | The lawn mower supports the stop command.            |
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)